### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by Jaypatel0812** [community/community#121296] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello , 
My github id was suspended 2 weeks ago. i try to contact support. but not getting any reply from them. is it possible to recover the account? 
please help me , because it's very important for me.

Thank you

**Comment by hassankhanmv** [comment]
hey there, Im also facing the same issue. I had 62 repository and its important for me. have you got any solution?

**Reply by Jaypatel0812** [reply]
not yet

**Comment by gurjeetsinghvirdee** [comment]
Hey @Jaypatel0812 @hassankhanmv 
I don't think you can recover the account, I didn't see any guy whose account was recovered as far as now!
I think you have to make a new one.

for next time you can make a backup of your code in google drive or any other place.

**Reply by Jaypatel0812** [reply]
thanks

**Comment by ghostinhershell** [comment]
Hi @Jaypatel0812,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by Jaypatel0812** [reply]
Ticket id: #2729665
Username : Jaypatel812
Email : [jay.khatara.455@gmail.com](mailto:jay.khatara.455@gmail.com)

Here is my details about my id . It's almost 2 weeks there is no reply from support team. Please help me !
